### ISSUE TITLE
The prediction is not well

### ISSUE THREAD

**Issue by Ariel-JUAN** [iro-cp/FCRN-DepthPrediction#39]
I restore the  model that you provide, and finetune the layers except the resnet50.
I use Berhu loss. And I use AdamOptimizer to minimize my loss, the learning rate is 0.0001, and  after 10000 steps the lr is 0.000001. The whole steps is 20000.
Here is my loss curve:
![finetune](https://user-images.githubusercontent.com/30546375/33521565-63fdbe5e-d810-11e7-9a6f-0b4db22af499.png)
Here is the prediction that I use my model:
![1](https://user-images.githubusercontent.com/30546375/33521569-76410594-d810-11e7-86ab-cb7e9c2438c8.png)
As you see, the loss value is small, but the prediction is not very well,  it is ambiguous. Can you give me some advice? Thanks!
@iro-cp

**Comment by harsanyika** [comment]
I experience a similar issue. Even when the losses seem relatively low (roughly the same as in the article), the images are still somewhat blurry, like the one above.

**Comment by Ariel-JUAN** [comment]
@harsanyika  Have you solved the issue?

**Comment by chrirupp** [comment]
Hi, I don't really understand your problem. The prediction looks ok to me. Not sure what you mean.

**Comment by Ariel-JUAN** [comment]
@chrirupp  The loss value is low, but compared to your model's prediction,  the result is not very good.
Here is the prediction used the model you trained:
![yuan](https://user-images.githubusercontent.com/30546375/33588230-ceac494e-d9ac-11e7-8c2e-98bdb2edd205.png)
Here is the prediction used the model I trained:
![shang](https://user-images.githubusercontent.com/30546375/33588244-de888288-d9ac-11e7-856c-ee5e4cc1c56c.png)
Have you seen the difference, I think it is just not as good as yours. Can you give me some advice?

**Comment by chrirupp** [comment]
Looking at one image is not conclusive. Predictions can be different but performance may be similar. The better way to compare is to use RMS or relative error on the whole test set. This give a much more objective view.

**Comment by Ariel-JUAN** [comment]
Thanks, I will try.

**Comment by Ariel-JUAN** [comment]
@chrirupp  Sorry to bother you again. I wonder when we train the model,  how to deal with the ground truth label? Do we need to resize them to (128, 160) and compare with predictions? Or we don't deal with the labels, and up-sample the predictions to (480, 640) ?
Thanks~

**Comment by chrirupp** [comment]
For training we down-sample the ground truth using nearest neighbor interpolation. For testing we up-sample the prediction to 640x480.

**Comment by Ariel-JUAN** [comment]
Thank you so much~

**Comment by Ariel-JUAN** [comment]
@chrirupp when we test metrics such as rel, do we need to use ground truth as images? Or we just pack the ground truth to the .mat, not the format of images. Thanks.

**Comment by chrirupp** [comment]
I am not sure what you are saying. You can find how to compute the relative error in our matlab code. How you store or load the data is up to you.

**Comment by luohongcheng** [comment]
@harsanyika Hi~ I am facing the same problem as you said. The Depth Prediction is blurry. Have you solved it?

**Comment by harsanyika** [comment]
Not really. My reverse Huber loss is lower than the reverse Huber loss of downloadable results (but I used more images + more epochs + online data augmentation). However, my relative loss and RMS loss is higher, and I still experience the blurriness.

I noticed that I reach my lowest loss sooner than suggested in the article (after 10-15 epoch). At that point, the LR is still high and as I lower the LR the network starts to overfit a little (I think). Right now I am unsure how to solve this problem.
